ToDoList/CartItem.swift

```diff
@@ -7,3 +7,34 @@
 
 import Foundation
 
+struct CartItem {
+    let name: String
+    let price: Int
+    let quantity: Int
+
+    /*
+      Function to calculate the total price of items in a shopping cart.
+      
+      - Parameters:
+          - items: An array of CartItem objects representing the items in the shopping cart.
+      
+      - Returns: A string indicating the total price, or a message if no items are found.
+      */
+     func calculatePrice(for items: [CartItem]) -> String {
+         if items.count == 0 {
+             return "No items found"
+         } else {
+             
+             var total = 0
+             items.forEach {
+                 total += $0.price
+             }
```

Comment:
Halo mas @malikghani ini bisa kita sederhanakan loh, pakai reduce.
```suggestion
             let total = items.map(\.price).reduce(0, +)
```

Reply:
Ini apakah time complexitynya lebih baik?